From: Rod Whitby <rod@whitby.id.au>
Subject: Disable broken DSM-G600 led support

LED support is not correct yet on DSM-G600, so disable it so it
doesn't do harm.

Signed-off-by: Rod Whitby <rod@whitby.id.au>
--

Index: apex/src/mach-ixp42x/dsmg600.h
===================================================================
--- apex.orig/src/mach-ixp42x/dsmg600.h	2008-05-28 10:53:47.000000000 +0930
+++ apex/src/mach-ixp42x/dsmg600.h	2008-05-28 10:53:55.000000000 +0930
@@ -52,8 +52,8 @@
 #define LEDe	((0<<1)|(1<<0)|(0<<2)|(0<<3))
 #define LEDf	((1<<1)|(1<<0)|(0<<2)|(0<<3))
 
-#define _L(l) MASK_AND_SET(GPIO_OUTR, LED_MASK, l)
-
+/* #define _L(l) MASK_AND_SET(GPIO_OUTR, LED_MASK, l) */
+#define _L(l) 0
 
 /* %%% FIXME %%% */
 #define GPIO_ER_OUTPUTS	 (1<<0)		/* Power LED */\
